src/WebJobs.Extensions.MobileApps/MobileAppsWebJobsStartup.cs

```diff
@@ -5,13 +5,13 @@
 using Microsoft.Azure.WebJobs.Hosting;
 using Microsoft.Extensions.Hosting;
 
-[assembly: WebJobsStartup(typeof(MobileAppsWebJobsStartup))]
+[assembly: WebJobsStartup(typeof(MobileAppsWebJobsStartup), "Mobile Apps")]
```

Comment:
I think we agreed that this name can include spaces, however for all the SDK core startup classes I didn't use spaces. Your call. E.g. EventHubs, ServiceBus, AzureStorage